coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -89,6 +83,25 @@ function MainNavigation() {
             linkText={"Audit log"}
           />
         </li>
+        <Box aria-hidden={"true"} paddingTop={"l1"} paddingBottom={"l2"}>
```

Comment:
Why is this aria-hidden? 🤔

Reply:
I copy pasted this markup from the other `Divider` we have in `MainNavigation`. Not sure why ^^

Maybe because it's a purely visual element that doesn't make sense to forward to a screen reader?

Reply:
Ah sorry, I needed to resize my window and change the view. The indentation looked off, that's why I didn't notice the box closed after the divider 🤦‍♀️  All good! :D